CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # CHANGELOG
 
+## v5.6.0 (2023-08-31)
+
+- Add ability to compare an EasyPost object to a parameter set object via `Matches` function
+  - Users can define a custom `Matches` function on any parameter set class
+- Fix parameter set serialization that was causing GetNextPage functions to malfunction
```

Comment:
I'd describe what about it wasn't working so people know if this affects them or not